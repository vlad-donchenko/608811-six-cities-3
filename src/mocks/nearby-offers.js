const nearbyOffers = [
  {
    bedrooms: 5,
    city: {
      location: {
        latitude: 52.38333,
        longitude: 4.9,
        zoom: 10
      },
      name: `Amsterdam`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`,
    goods: [`Wi-Fi`, `Heating`, `Kitchen`, `Fridge`, `Washing machine`, `Baby seat`],
    host: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 50,
      isPro: true,
      name: `Angelina`
    },
    id: 50,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: true,
    location: {
      latitude: 52.398298,
      longitude: 4.848328,
      zoom: 8
    },
    maxAdults: 3,
    previewImage: `apartment-01.jpg`,
    price: 424,
    rating: 4,
    title: `Beautiful & luxurious apartment at great location - Nearby`,
    type: `Private room`
  },
  {
    bedrooms: 5,
    city: {
      location: {
        latitude: 52.38333,
        longitude: 4.9,
        zoom: 10
      },
      name: `Amsterdam`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`,
    goods: [`Wi-Fi`, `Heating`, `Kitchen`, `Fridge`, `Washing machine`, `Baby seat`],
    host: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 51,
      isPro: true,
      name: `Angelina`
    },
    id: 51,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: false,
    location: {
      latitude: 52.397288,
      longitude: 4.852012,
      zoom: 8
    },
    maxAdults: 3,
    previewImage: `apartment-01.jpg`,
    price: 124,
    rating: 2,
    title: `Beautiful & luxurious apartment at great location - Nearby`,
    type: `Private room`
  },
  {
    bedrooms: 5,
    city: {
      location: {
        latitude: 52.38333,
        longitude: 4.9,
        zoom: 10
      },
      name: `Amsterdam`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`,
    goods: [`Wi-Fi`, `Heating`, `Kitchen`, `Fridge`, `Washing machine`, `Baby seat`],
    host: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 52,
      isPro: true,
      name: `Angelina`
    },
    id: 52,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: true,
    isPremium: true,
    location: {
      latitude: 52.395049,
      longitude: 4.854962,
      zoom: 8
    },
    maxAdults: 3,
    previewImage: `apartment-02.jpg`,
    price: 924,
    rating: 2,
    title: `Beautiful & luxurious apartment at great location - Nearby`,
    type: `Private room`
  },
  {
    bedrooms: 3,
    city: {
      location: {
        latitude: 48.866411,
        longitude: 2.331896,
        zoom: 15
      },
      name: `Paris`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Wi-Fi`, `Kitchen`, `Towels`, `Baby seat`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 53,
      isPro: true,
      name: `Max`
    },
    id: 53,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: false,
    location: {
      latitude: 48.869508,
      longitude: 2.327385,
      zoom: 8
    },
    maxAdults: 4,
    previewImage: `apartment-02.jpg`,
    price: 1100,
    rating: 1,
    title: `Sofitel Paris Le Faubourg - Nearby`,
    type: `Apartment`
  },
  {
    bedrooms: 3,
    city: {
      location: {
        latitude: 48.866411,
        longitude: 2.331896,
        zoom: 15
      },
      name: `Paris`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Wi-Fi`, `Kitchen`, `Towels`, `Baby seat`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 54,
      isPro: true,
      name: `Max`
    },
    id: 54,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: true,
    isPremium: true,
    location: {
      latitude: 48.870955,
      longitude: 2.330679,
      zoom: 8
    },
    maxAdults: 4,
    previewImage: `apartment-02.jpg`,
    price: 100,
    rating: 5,
    title: `Sofitel Paris Le Faubourg - Nearby`,
    type: `Apartment`
  },
  {
    bedrooms: 3,
    city: {
      location: {
        latitude: 48.866411,
        longitude: 2.331896,
        zoom: 15
      },
      name: `Paris`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Wi-Fi`, `Kitchen`, `Towels`, `Baby seat`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 55,
      isPro: true,
      name: `Max`
    },
    id: 55,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: true,
    location: {
      latitude: 48.869918,
      longitude: 2.327986,
      zoom: 8
    },
    maxAdults: 4,
    previewImage: `apartment-03.jpg`,
    price: 190,
    rating: 3,
    title: `Sofitel Paris Le Faubourg - Nearby`,
    type: `Apartment`
  }
];

export default nearbyOffers;
