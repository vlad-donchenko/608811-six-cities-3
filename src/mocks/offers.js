import {getRandomInteger} from "../utils/index";

const INDICATOR = 0.5;

const titles = [
  `Beautiful & luxurious apartment at great location`,
  `Wood and stone place`,
  `Canal View Prinsengracht`,
  `Nice, cozy, warm big bed apartment`,
  `Wood and stone place`
];

const images = [
  `room.jpg`,
  `apartment-02.jpg`,
  `apartment-03.jpg`,
  `apartment-01.jpg`,
  `apartment-01.jpg`,
];

const goods = [
  `Wi-Fi`,
  `Heating`,
  `Kitchen`,
  `Fridge`,
  `Washing machine`,
  `Coffee machine`,
  `Dishwasher`,
  `Towels`,
  `Baby seat`,
  `Cabel TV`
];

const names = [
  `Angelina`,
  `Max`,
  `Sam`,
  `Nick`,
  `Jane`
];

const avatars = [
  `avatar-angelina.jpg`,
  `avatar-max.jpg`
];

const descriptions = [
  `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`,
  `An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful`,
  `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`
];

const latitudes = [
  52.3909553943508,
  52.369553943508,
  52.3909553943508,
  52.3809553943508,
  52.3709553943508
];

const longitude = [
  4.85309666406198,
  4.85309666406198,
  4.929309666406198,
  4.939309666406198,
  4.979309666406198
];

const getGoods = () => goods.filter(() => Math.random() > INDICATOR);

const getDescription = () => {
  return descriptions.filter(() => {
    return Math.random() > INDICATOR;
  }).join(`. `);
};

const generateOffers = () => {
  return titles.map((title, id) => {
    return {
      bedrooms: getRandomInteger(1, 7),
      city: {
        location: {
          latitude: 52.38333,
          longitude: 4.9,
          zoom: 10,
        },
        name: `Amsterdam`
      },
      description: getDescription(),
      goods: getGoods(),
      host: {
        avatarUrl: avatars[getRandomInteger(0, avatars.length - 1)],
        id,
        isPro: Math.random() > INDICATOR,
        name: names[id]
      },
      id,
      images,
      isFavorite: Math.random() > INDICATOR,
      isPremium: Math.random() > INDICATOR,
      location: {
        latitude: latitudes[id],
        longitude: longitude[id],
        zoom: 8
      },
      maxAdults: getRandomInteger(1, 7),
      previewImage: images[getRandomInteger(0, images.length - 1)],
      price: getRandomInteger(100, 500),
      rating: getRandomInteger(0, 5),
      title,
      type: Math.random() > INDICATOR ? `Private room` : `Apartment`,
    };
  });
};

const offers = generateOffers();

export {offers, names};
