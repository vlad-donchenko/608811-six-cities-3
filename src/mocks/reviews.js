const reviews = [
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:56.569Z`,
    id: 1,
    rating: 4,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 1,
      isPro: false,
      name: `Max`
    }
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-06-08T14:13:56.569Z`,
    id: 2,
    rating: 5,
    user: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 2,
      isPro: true,
      name: `Angelina`
    }
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-09-08T14:13:56.569Z`,
    id: 3,
    rating: 1,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 3,
      isPro: false,
      name: `Max`
    }
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-10-08T14:13:56.569Z`,
    id: 4,
    rating: 5,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 4,
      isPro: false,
      name: `Angelina`
    }
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-10-08T14:13:56.569Z`,
    id: 5,
    rating: 2,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 3,
      isPro: true,
      name: `Max`
    }
  }
];

export {reviews};
