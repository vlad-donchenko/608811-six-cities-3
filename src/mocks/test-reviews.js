const reviews = [
  {
    comment: `Lorem ipsum dolor sit amet, consectetur adipisicing elit.`,
    date: `2019-08-08T14:13:56.569Z`,
    id: 1,
    rating: 5,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 1,
      isPro: true,
      name: `Max`
    }
  },
  {
    comment: `Lorem ipsum dolor sit amet, consectetur adipisicing elit.`,
    date: `2019-06-03T14:13:56.569Z`,
    id: 2,
    rating: 4,
    user: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 2,
      isPro: false,
      name: `Angelina`
    }
  },
  {
    comment: `Lorem ipsum dolor sit amet, consectetur adipisicing elit.`,
    date: `2019-06-01T14:13:56.569Z`,
    id: 3,
    rating: 2,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 3,
      isPro: false,
      name: `Max`
    }
  },
  {
    comment: `Lorem ipsum dolor sit amet, consectetur adipisicing elit.`,
    date: `2019-11-08T14:13:56.569Z`,
    id: 4,
    rating: 2,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 4,
      isPro: false,
      name: `Angelina`
    }
  },
  {
    comment: `Lorem ipsum dolor sit amet, consectetur adipisicing elit.`,
    date: `2019-01-08T14:13:56.569Z`,
    id: 5,
    rating: 4,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 3,
      isPro: false,
      name: `Max`
    }
  }
];

export {reviews};
