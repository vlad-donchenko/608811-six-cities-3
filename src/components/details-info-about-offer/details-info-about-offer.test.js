import React from "react";
import renderer from "react-test-renderer";
import DetailsInfoAboutOffer from "./details-info-about-offer";
import {offers} from "../../mocks/test-offer";

it(`Should render <DetailsInfoAboutOffer/>`, () => {
  const tree = renderer.create(<DetailsInfoAboutOffer offer={offers[0]}/>).toJSON();

  expect(tree).toMatchSnapshot();
});
