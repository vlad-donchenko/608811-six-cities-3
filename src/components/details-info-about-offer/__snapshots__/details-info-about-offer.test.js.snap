// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render <DetailsInfoAboutOffer/> 1`] = `
<div
  className="page"
>
  <header
    className="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-link"
            href="main.html"
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            >
              <a
                className="header__nav-link header__nav-link--profile"
                href="#"
              >
                <div
                  className="header__avatar-wrapper user__avatar-wrapper"
                />
                <span
                  className="header__user-name user__name"
                >
                  Oliver.conner@gmail.com
                </span>
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <main
    className="page__main page__main--property"
  >
    <section
      className="property"
    >
      <div
        className="property__gallery-container container"
      >
        <div
          className="property__gallery"
        >
          <div
            className="property__image-wrapper"
          >
            <img
              alt="Photo studio"
              className="property__image"
              src="img/room.jpg"
            />
          </div>
          <div
            className="property__image-wrapper"
          >
            <img
              alt="Photo studio"
              className="property__image"
              src="img/apartment-02.jpg"
            />
          </div>
          <div
            className="property__image-wrapper"
          >
            <img
              alt="Photo studio"
              className="property__image"
              src="img/apartment-03.jpg"
            />
          </div>
          <div
            className="property__image-wrapper"
          >
            <img
              alt="Photo studio"
              className="property__image"
              src="img/apartment-01.jpg"
            />
          </div>
          <div
            className="property__image-wrapper"
          >
            <img
              alt="Photo studio"
              className="property__image"
              src="img/apartment-01.jpg"
            />
          </div>
        </div>
      </div>
      <div
        className="property__container container"
      >
        <div
          className="property__wrapper"
        >
          
          <div
            className="property__name-wrapper"
          >
            <h1
              className="property__name"
            >
              Beautiful & luxurious studio at great location - 1
            </h1>
            <button
              className="property__bookmark-button button "
              type="button"
            >
              <svg
                className="property__bookmark-icon"
                height="33"
                width="31"
              >
                <use
                  xlinkHref="#icon-bookmark"
                />
              </svg>
              <span
                className="visually-hidden"
              >
                To bookmarks
              </span>
            </button>
          </div>
          <div
            className="property__rating rating"
          >
            <div
              className="property__stars rating__stars"
            >
              <span
                style={
                  Object {
                    "width": "96%",
                  }
                }
              />
              <span
                className="visually-hidden"
              >
                Rating
              </span>
            </div>
            <span
              className="property__rating-value rating__value"
            >
              4.8
            </span>
          </div>
          <ul
            className="property__features"
          >
            <li
              className="property__feature property__feature--entire"
            >
              apartment
            </li>
            <li
              className="property__feature property__feature--bedrooms"
            >
              3
               Bedrooms
            </li>
            <li
              className="property__feature property__feature--adults"
            >
              Max 
              4
               adults
            </li>
          </ul>
          <div
            className="property__price"
          >
            <b
              className="property__price-value"
            >
              €
              120
            </b>
            <span
              className="property__price-text"
            >
               night
            </span>
          </div>
          <div
            className="property__inside"
          >
            <h2
              className="property__inside-title"
            >
              What’s inside
            </h2>
            <ul
              className="property__inside-list"
            >
              <li
                className="property__inside-item"
              >
                good
              </li>
              <li
                className="property__inside-item"
              >
                good
              </li>
              <li
                className="property__inside-item"
              >
                good
              </li>
              <li
                className="property__inside-item"
              >
                good
              </li>
              <li
                className="property__inside-item"
              >
                good
              </li>
              <li
                className="property__inside-item"
              >
                good
              </li>
            </ul>
          </div>
          <div
            className="property__host"
          >
            <h2
              className="property__host-title"
            >
              Meet the host
            </h2>
            <div
              className="property__host-user user"
            >
              <div
                className="property__avatar-wrapper property__avatar-wrapper--pro user__avatar-wrapper"
              >
                <img
                  alt="Host avatar"
                  className="property__avatar user__avatar"
                  height="74"
                  src="img/avatar-angelina.jpg"
                  width="74"
                />
              </div>
              <span
                className="property__user-name"
              >
                Angelina
              </span>
            </div>
            <div
              className="property__description"
            >
              <p
                className="property__text"
              >
                A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.
              </p>
            </div>
          </div>
          <section
            className="property__reviews reviews"
          >
            <h2
              className="reviews__title"
            >
              Reviews · 
              <span
                className="reviews__amount"
              >
                5
              </span>
            </h2>
            <ul
              className="reviews__list"
            >
              <li
                className="reviews__item"
              >
                <div
                  className="reviews__user user"
                >
                  <div
                    className="reviews__avatar-wrapper user__avatar-wrapper"
                  >
                    <img
                      alt="Reviews avatar"
                      className="reviews__avatar user__avatar"
                      height="54"
                      src="img/avatar-max.jpg"
                      width="54"
                    />
                  </div>
                  <span
                    className="reviews__user-name"
                  >
                    Angelina
                  </span>
                </div>
                <div
                  className="reviews__info"
                >
                  <div
                    className="reviews__rating rating"
                  >
                    <div
                      className="reviews__stars rating__stars"
                    >
                      <span
                        style={
                          Object {
                            "width": "100%",
                          }
                        }
                      />
                      <span
                        className="visually-hidden"
                      >
                        5
                      </span>
                    </div>
                  </div>
                  <p
                    className="reviews__text"
                  >
                    A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.
                  </p>
                  <time
                    className="reviews__time"
                    dateTime="2019-09-08"
                  >
                    October 8 2019
                  </time>
                </div>
              </li>
              <li
                className="reviews__item"
              >
                <div
                  className="reviews__user user"
                >
                  <div
                    className="reviews__avatar-wrapper user__avatar-wrapper"
                  >
                    <img
                      alt="Reviews avatar"
                      className="reviews__avatar user__avatar"
                      height="54"
                      src="img/avatar-max.jpg"
                      width="54"
                    />
                  </div>
                  <span
                    className="reviews__user-name"
                  >
                    Max
                  </span>
                </div>
                <div
                  className="reviews__info"
                >
                  <div
                    className="reviews__rating rating"
                  >
                    <div
                      className="reviews__stars rating__stars"
                    >
                      <span
                        style={
                          Object {
                            "width": "40%",
                          }
                        }
                      />
                      <span
                        className="visually-hidden"
                      >
                        2
                      </span>
                    </div>
                  </div>
                  <p
                    className="reviews__text"
                  >
                    A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.
                  </p>
                  <time
                    className="reviews__time"
                    dateTime="2019-09-08"
                  >
                    October 8 2019
                  </time>
                </div>
              </li>
              <li
                className="reviews__item"
              >
                <div
                  className="reviews__user user"
                >
                  <div
                    className="reviews__avatar-wrapper user__avatar-wrapper"
                  >
                    <img
                      alt="Reviews avatar"
                      className="reviews__avatar user__avatar"
                      height="54"
                      src="img/avatar-max.jpg"
                      width="54"
                    />
                  </div>
                  <span
                    className="reviews__user-name"
                  >
                    Max
                  </span>
                </div>
                <div
                  className="reviews__info"
                >
                  <div
                    className="reviews__rating rating"
                  >
                    <div
                      className="reviews__stars rating__stars"
                    >
                      <span
                        style={
                          Object {
                            "width": "20%",
                          }
                        }
                      />
                      <span
                        className="visually-hidden"
                      >
                        1
                      </span>
                    </div>
                  </div>
                  <p
                    className="reviews__text"
                  >
                    A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.
                  </p>
                  <time
                    className="reviews__time"
                    dateTime="2019-08-08"
                  >
                    September 8 2019
                  </time>
                </div>
              </li>
              <li
                className="reviews__item"
              >
                <div
                  className="reviews__user user"
                >
                  <div
                    className="reviews__avatar-wrapper user__avatar-wrapper"
                  >
                    <img
                      alt="Reviews avatar"
                      className="reviews__avatar user__avatar"
                      height="54"
                      src="img/avatar-angelina.jpg"
                      width="54"
                    />
                  </div>
                  <span
                    className="reviews__user-name"
                  >
                    Angelina
                  </span>
                </div>
                <div
                  className="reviews__info"
                >
                  <div
                    className="reviews__rating rating"
                  >
                    <div
                      className="reviews__stars rating__stars"
                    >
                      <span
                        style={
                          Object {
                            "width": "100%",
                          }
                        }
                      />
                      <span
                        className="visually-hidden"
                      >
                        5
                      </span>
                    </div>
                  </div>
                  <p
                    className="reviews__text"
                  >
                    A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.
                  </p>
                  <time
                    className="reviews__time"
                    dateTime="2019-05-08"
                  >
                    June 8 2019
                  </time>
                </div>
              </li>
              <li
                className="reviews__item"
              >
                <div
                  className="reviews__user user"
                >
                  <div
                    className="reviews__avatar-wrapper user__avatar-wrapper"
                  >
                    <img
                      alt="Reviews avatar"
                      className="reviews__avatar user__avatar"
                      height="54"
                      src="img/avatar-max.jpg"
                      width="54"
                    />
                  </div>
                  <span
                    className="reviews__user-name"
                  >
                    Max
                  </span>
                </div>
                <div
                  className="reviews__info"
                >
                  <div
                    className="reviews__rating rating"
                  >
                    <div
                      className="reviews__stars rating__stars"
                    >
                      <span
                        style={
                          Object {
                            "width": "80%",
                          }
                        }
                      />
                      <span
                        className="visually-hidden"
                      >
                        4
                      </span>
                    </div>
                  </div>
                  <p
                    className="reviews__text"
                  >
                    A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.
                  </p>
                  <time
                    className="reviews__time"
                    dateTime="2019-04-08"
                  >
                    May 8 2019
                  </time>
                </div>
              </li>
            </ul>
          </section>
        </div>
      </div>
      <section
        className="property__map map"
      >
        <div
          className="map__content"
          id="map"
        />
      </section>
    </section>
    <div
      className="container"
    >
      <section
        className="near-places places"
      >
        <h2
          className="near-places__title"
        >
          Other places in the neighbourhood
        </h2>
        <div
          className="near-places__list places__list"
        >
          <article
            className="near-places__card place-card"
            onMouseOver={[Function]}
          >
            <div
              className="place-card__mark"
            >
              <span>
                Premium
              </span>
            </div>
            <div
              className="near-places__card__image-wrapper place-card__image-wrapper"
            >
              <a
                href="#"
              >
                <img
                  alt="Beautiful & luxurious apartment at great location"
                  className="place-card__image"
                  height="200"
                  src="img/apartment-01.jpg"
                  width="260"
                />
              </a>
            </div>
            <div
              className="place-card__info"
            >
              <div
                className="place-card__price-wrapper"
              >
                <div
                  className="place-card__price"
                >
                  <b
                    className="place-card__price-value"
                  >
                    €
                    100
                  </b>
                  <span
                    className="place-card__price-text"
                  >
                    / night
                  </span>
                </div>
                <button
                  className="place-card__bookmark-button button place-card__bookmark-button--active"
                  type="button"
                >
                  <svg
                    className="place-card__bookmark-icon"
                    height="19"
                    width="18"
                  >
                    <use
                      xlinkHref="#icon-bookmark"
                    />
                  </svg>
                  <span
                    className="visually-hidden"
                  >
                    To bookmarks
                  </span>
                </button>
              </div>
              <div
                className="place-card__rating rating"
              >
                <div
                  className="place-card__stars rating__stars"
                >
                  <span
                    style={
                      Object {
                        "width": "0%",
                      }
                    }
                  />
                  <span
                    className="visually-hidden"
                  >
                    Rating
                  </span>
                </div>
              </div>
              <h2
                className="place-card__name"
              >
                <a
                  href="#"
                  onClick={[Function]}
                >
                  Beautiful & luxurious apartment at great location
                </a>
              </h2>
              <p
                className="place-card__type"
              >
                Private room
              </p>
            </div>
          </article>
          <article
            className="near-places__card place-card"
            onMouseOver={[Function]}
          >
            <div
              className="place-card__mark"
            >
              <span>
                Premium
              </span>
            </div>
            <div
              className="near-places__card__image-wrapper place-card__image-wrapper"
            >
              <a
                href="#"
              >
                <img
                  alt="Wood and stone place"
                  className="place-card__image"
                  height="200"
                  src="img/apartment-02.jpg"
                  width="260"
                />
              </a>
            </div>
            <div
              className="place-card__info"
            >
              <div
                className="place-card__price-wrapper"
              >
                <div
                  className="place-card__price"
                >
                  <b
                    className="place-card__price-value"
                  >
                    €
                    238
                  </b>
                  <span
                    className="place-card__price-text"
                  >
                    / night
                  </span>
                </div>
                <button
                  className="place-card__bookmark-button button "
                  type="button"
                >
                  <svg
                    className="place-card__bookmark-icon"
                    height="19"
                    width="18"
                  >
                    <use
                      xlinkHref="#icon-bookmark"
                    />
                  </svg>
                  <span
                    className="visually-hidden"
                  >
                    To bookmarks
                  </span>
                </button>
              </div>
              <div
                className="place-card__rating rating"
              >
                <div
                  className="place-card__stars rating__stars"
                >
                  <span
                    style={
                      Object {
                        "width": "20%",
                      }
                    }
                  />
                  <span
                    className="visually-hidden"
                  >
                    Rating
                  </span>
                </div>
              </div>
              <h2
                className="place-card__name"
              >
                <a
                  href="#"
                  onClick={[Function]}
                >
                  Wood and stone place
                </a>
              </h2>
              <p
                className="place-card__type"
              >
                Private room
              </p>
            </div>
          </article>
          <article
            className="near-places__card place-card"
            onMouseOver={[Function]}
          >
            <div
              className="place-card__mark"
            >
              <span>
                Premium
              </span>
            </div>
            <div
              className="near-places__card__image-wrapper place-card__image-wrapper"
            >
              <a
                href="#"
              >
                <img
                  alt="Canal View Prinsengracht"
                  className="place-card__image"
                  height="200"
                  src="img/apartment-02.jpg"
                  width="260"
                />
              </a>
            </div>
            <div
              className="place-card__info"
            >
              <div
                className="place-card__price-wrapper"
              >
                <div
                  className="place-card__price"
                >
                  <b
                    className="place-card__price-value"
                  >
                    €
                    375
                  </b>
                  <span
                    className="place-card__price-text"
                  >
                    / night
                  </span>
                </div>
                <button
                  className="place-card__bookmark-button button place-card__bookmark-button--active"
                  type="button"
                >
                  <svg
                    className="place-card__bookmark-icon"
                    height="19"
                    width="18"
                  >
                    <use
                      xlinkHref="#icon-bookmark"
                    />
                  </svg>
                  <span
                    className="visually-hidden"
                  >
                    To bookmarks
                  </span>
                </button>
              </div>
              <div
                className="place-card__rating rating"
              >
                <div
                  className="place-card__stars rating__stars"
                >
                  <span
                    style={
                      Object {
                        "width": "20%",
                      }
                    }
                  />
                  <span
                    className="visually-hidden"
                  >
                    Rating
                  </span>
                </div>
              </div>
              <h2
                className="place-card__name"
              >
                <a
                  href="#"
                  onClick={[Function]}
                >
                  Canal View Prinsengracht
                </a>
              </h2>
              <p
                className="place-card__type"
              >
                Apartment
              </p>
            </div>
          </article>
        </div>
      </section>
    </div>
  </main>
</div>
`;
